concent_api/core/tests/test_core_views.py

```diff
@@ -0,0 +1,74 @@
+        assert Message.objects.count()       == 0
+        assert MessageStatus.objects.count() == 0
+
+        response = self.client.post(reverse('core:send'), data = json.dumps(self.correct_data), content_type = 'application/json', HTTP_CONCENT_CLIENT_PUBLIC_KEY = self.public_key)
+
+        self.assertEqual(response.status_code, 202)  # pylint: disable=no-member
+        self.assertEqual(len(Message.objects.all()),       1)
+        self.assertEqual(len(MessageStatus.objects.all()), 1)
+
+    @freeze_time("2017-11-17 10:00:00")
+    def test_send_should_return_http_200_if_message_timeout(self):
+        self.assertEqual(len(Message.objects.all()), 0)
+        self.assertEqual(len(MessageStatus.objects.all()), 0)
+
+        self.correct_data['message_task_to_compute']['deadline'] = 1510911000
+        response = self.client.post("/api/v1/send/", data = json.dumps(self.correct_data), content_type = 'application/json', concent_client_public_key = self.public_key, HTTP_CONCENT_CLIENT_PUBLIC_KEY = self.public_key)
+
+        self.assertEqual(response.status_code, 200)  # pylint: disable=no-member
+        self.assertEqual(response.json()['type'], 'MessageRejectReportComputedTask')  # pylint: disable=no-member
+
+    @freeze_time("2017-11-17 10:00:00")
+    def test_send_should_return_http_400_if_data_is_incorrect(self):
+        data = {
+            "type":      "MessageForceReportComputedTask",
+            "timestamp": 1510911047,
+            "message_task_to_compute":      {
+                "type": "MessageTaskToCompute"
+            }
+        }
+
+        response = self.client.post(reverse('core:send'), data = json.dumps(data), content_type = 'application/json', concent_client_public_key = self.public_key, HTTP_CONCENT_CLIENT_PUBLIC_KEY = self.public_key)
+
+        self.assertEqual(response.status_code, 400)  # pylint: disable=no-member
+        self.assertEqual(json.loads(response.content.decode('utf-8'))['error'], "time limit doesn't exist Inside message task.")  # pylint: disable=no-member
+
+        self.correct_data['message_task_to_compute']['deadline'] = "1510909200"
+        response = self.client.post(reverse('core:send'), data = json.dumps(self.correct_data), content_type = 'application/json', concent_client_public_key = self.public_key, HTTP_CONCENT_CLIENT_PUBLIC_KEY = self.public_key)
+
+        self.assertEqual(response.status_code, 400)  # pylint: disable=no-member
+        self.assertEqual(json.loads(response.content.decode('utf-8'))['error'], "Wrong type of inside message task time limit. Not an integer.")  # pylint: disable=no-member
+
+    @freeze_time("2017-11-17 10:00:00")
+    def test_send_should_return_http_400_if_task_id_already_use(self):
+        response = self.client.post(reverse('core:send'), data = json.dumps(self.correct_data), content_type = 'application/json', concent_client_public_key = self.public_key, HTTP_CONCENT_CLIENT_PUBLIC_KEY = self.public_key)
```

Comment:
You should check the response of this first call too. And use different variable names for these two responses.